src/libs/Wallet/RNAddToWalletButton/index.native.tsx

```diff
@@ -0,0 +1,63 @@
+import {AddToWalletButton} from '@expensify/react-native-wallet';
+import React, {useCallback, useEffect} from 'react';
+import {View} from 'react-native';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import {checkIfWalletIsAvailable, handleAddCardToWallet, isCardInWallet} from '..';
+import type AddToWalletButtonProps from './types';
+
+function RNAddToWalletButton({card, cardHolderName, buttonStyle}: AddToWalletButtonProps) {
```

Comment:
Not sure if we need the RN here
```suggestion
function AddToWalletButton({card, cardHolderName, buttonStyle}: AddToWalletButtonProps) {
```

Reply:
probably interferes with the import, but perhaps we could alias the import as import { AddToWalletButton as RNWAddToWalletButton }?